packages/frontend/src/app/search/search.component.ts

```diff
@@ -71,15 +99,25 @@ export class SearchComponent implements OnInit {
   }
 
   onEnterKeyUp(option: string): void {
-    this.searchOptionSelected.emit(this.interpreter.parseSearchInput(option));
+    this.launchSearch(this.interpreter.parseSearchInput(option));
```

Comment:
The search bar handles the redirection itself instead of moving the responsibility to the parent component.